lib/trino-hive-formats/src/main/java/io/trino/hive/formats/line/json/JsonDeserializer.java

```diff
@@ -382,7 +380,12 @@ public RealDecoder()
         void decodeValue(JsonParser parser, BlockBuilder builder)
                 throws IOException
         {
-            REAL.writeLong(builder, floatToRawIntBits(parser.getFloatValue()));
+            try {
+                REAL.writeLong(builder, floatToRawIntBits(parser.getFloatValue()));
+            }
+            catch (IOException e) {
```

Comment:
I think this should just be `JsonParseException` (which is a subclass of `IOException`). The other decoder values should follow the same logic too.